packages/admin-next/dashboard/src/v2-routes/orders/order-detail/constants.ts

```diff
@@ -0,0 +1,30 @@
+const DEFAULT_PROPERTIES = [
+  "id",
+  "status",
+  "created_at",
+  "email",
+  // "fulfillment_status", // -> TODO replacement for this
+  // "payment_status", // -> TODO replacement for this
+  "display_id",
+  "currency_code",
+  // --- TOTALS ---
+  "total",
+  "subtotal",
+  "discounts_total",
+  "shipping_total",
+  "tax_total",
+]
+
+const DEFAULT_RELATIONS = [
+  "*customer",
+  "*items", // -> we get LineItem here with added `quantity` and `detail` which is actually an OrderItem (which is a parent object to LineItem in the DB)
+  // "*items.*variant", // -> TODO should we link this?
```

Comment:
@carlos-r-l-rodrigues, I guess this will need a similar link as to cart items

Reply:
Yes, the same definition just changing CART to ORDER